gcp/shared/utils/config_models.py

```
@@ -164,6 +166,48 @@ class FollowupSurveysConfig(BaseModel):
+                raise ValueError(
+                    f"shift '{key}' must have exactly 3 times, got {len(times)}"
+                )
+            for t in times:
+                if not hhmm.match(t):
+                    raise ValueError(
+                        f"shift '{key}' time '{t}' must match HH:MM"
+                    )
+        return v
+
+    @model_validator(mode="after")
+    def default_shift_in_shifts(self) -> "ShiftTimesConfig":
```

Ruff (0.15.15)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)